core/trino-main/src/test/java/io/trino/sql/gen/BenchmarkInCodeGenerator.java

```diff
@@ -83,31 +87,43 @@
         public void setup()
         {
             Random random = new Random();
-            RowExpression[] arguments = new RowExpression[1 + inListCount];
```

Comment:
It seems that we should also parametrize hit ratio. How often do we hit list in current benchmark?

Reply:
I added parametrization on that. 
With current code overlap of page and in list was ~0